src/VisualStudio/IntegrationTest/TestUtilities/DialogHelpers.cs

```diff
@@ -35,6 +35,17 @@ public static AutomationElement FindDialog(int visualStudioHWnd, string dialogAu
         }
 
         /// <summary>
+        /// Used to find legacy dialogs that don't have an AutomationId
```

Comment:
Now that we have one for each of `AutomationId` and `AutomationName`, you could make the world a much better place by fixing up the regular `FindDialog` parameter names to say `Id` instead of `Name`, and maybe rename them to be `FindDialogById`?